app/views/fridges/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= @fridge.name %>
+<ul>
+  <% @ingredients.each do |ingredient| %>
```

Comment:
this is just a question for me to understand: should it not be "fridge_ingredients" instead of just ingredients?

Reply:
never mind, you did that in the show method, right? linking the ingredients to the fridge ingredients.

Reply:
You are correct, it is the fridge_ingredients. This one is still in progress, The main point was devise, to set the fridge as default per user. so them I'll work and show. so for now it can be this way.

Reply:
okay, fine by me. maybe make a comment there so we are aware of it.

Reply:
comment added :)